Makefile

```diff
@@ -112,10 +112,7 @@ docker-build: ## Build docker image with the manager.
 
 docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
-docker-tag-base:
-	docker tag 	${IMG} ${IMAGE_TAG_BASE}:${BASE_VERSION}
-docker-push-base:
```

Comment:
These were not used as far as I could tell, and were confusing, so I removed them